vms/platformvm/vm.go

```diff
@@ -111,14 +105,9 @@ func (vm *VM) Initialize(
 ) error {
 	chainCtx.Log.Verbo("initializing platform chain")
 
-	platformConfig := Config{}
-	if len(configBytes) > 0 {
-		if err := stdjson.Unmarshal(configBytes, &platformConfig); err != nil {
-			return err
-		}
-		chainCtx.Log.Info("VM config initialized",
-			zap.Reflect("config", platformConfig),
-		)
```

Comment:
nit: can we still log the config we are initializing with? I know before we only logged it if a config was provided... But I think it's fine to always log it (seems to flow more with the code as it is now)

Reply:
done